code/modules/projectiles/projectile/special/goat.dm

```diff
@@ -0,0 +1,18 @@
+/obj/projectile/goat
+	name = "goat"
+	icon = 'icons/mob/animal.dmi'
+	icon_state = "goat"
+	damage = 20
+	damage_type = BRUTE
+	flag = "bullet"
+
+/obj/projectile/goat/on_hit(atom/target)
+	if(target == firer)
+		forceMove(target.loc)
+		return
+	paralyze = 20
```

Comment:
why are these defined in /obj/projectile/goat/on_hit(atom/target) instead of in /obj/projectile/goat?

also, this is a ranged hardstun weapon with a lot of ammo that deals 20 brute damage per shot and self-recharges. it's like a meathook on steroids. the only downside to it is that your own goats will sometimes get in the way of your shots. why did you think that this was a good item to try to add to the game?